src/main/kotlin/com/sourcegraph/cody/chat/SignInWithSourcegraphPanel.kt

```diff
@@ -25,42 +25,44 @@ import com.sourcegraph.cody.ui.UnderlinedActionLink
 import java.awt.BorderLayout
 import java.awt.FlowLayout
 import java.awt.GridBagLayout
+import java.awt.event.ActionEvent
 import java.awt.event.ActionListener
 import javax.swing.BoxLayout
+import javax.swing.JButton
 import javax.swing.JLabel
 import javax.swing.JPanel
 import javax.swing.border.Border
 
 class SignInWithSourcegraphPanel(private val project: Project) : JPanel() {
 
-  private val mainButton = UIComponents.createMainButton("Sign in for free with Sourcegraph.com")
+  private val signInWithGithubButton =
+      UIComponents.createMainButton("Sign in with GitHub", Icons.SignIn.Github)
```

Comment:
I see that for now "Sign in with GitHub" is used twice in this diff. 
Would it be possible to bring the texts into `SsoAuthMethod` enum?

(the same concern about all `"Sign in with .*"` ofc 😃)